modules/alloy/src/main/ts/ephox/alloy/behaviour/sandboxing/SandboxApis.ts

```diff
@@ -54,19 +52,11 @@ const close = (sandbox: AlloyComponent, sConfig: SandboxingConfig, sState: Sandb
   });
 };
 
-const isOpen = (_sandbox: AlloyComponent, _sConfig: SandboxingConfig, sState: SandboxingState) => {
-  return sState.isOpen();
-};
+const isOpen = (_sandbox: AlloyComponent, _sConfig: SandboxingConfig, sState: SandboxingState) => sState.isOpen();
 
-const isPartOf = (sandbox: AlloyComponent, sConfig: SandboxingConfig, sState: SandboxingState, queryElem: Element) => {
-  return isOpen(sandbox, sConfig, sState) && sState.get().exists((data) => {
-    return sConfig.isPartOf(sandbox, data, queryElem);
-  });
-};
+const isPartOf = (sandbox: AlloyComponent, sConfig: SandboxingConfig, sState: SandboxingState, queryElem: Element) => isOpen(sandbox, sConfig, sState) && sState.get().exists((data) => sConfig.isPartOf(sandbox, data, queryElem));
```

Comment:
Another one where it's all been shoved onto a single line.

Reply:
Fixed